Sources/StreamChatUI/Components.swift

```diff
@@ -123,6 +123,9 @@ public struct Components {
     /// Note: This is currently an experimental feature that we are actively
     /// working on and testing to make sure it is stable.
     public var _messageListDiffingEnabled = false
+    
+    /// A boolean value that determines whether Markdown is active for messages to be formatted.
+    public var markdownFormatterEnabled = true
```

Comment:
Better to move this flag close the formatter

Reply:
Done